cli.js

```diff
@@ -1103,20 +1103,34 @@ cli.progress = function (progress, decimals, stream) {
     }
     last_progress_call = now;
 
-
+    var pwr = Math.pow(10, decimals);
+    var percentageAsNum = Math.floor(progress * 100 * pwr) / pwr; 
```

Comment:
In my defense, I was trying my best to make sense of the casing scheme in the file. Best I could surmise was that global vars tended to have underscores and local vars tended to be camel case. Still, sorry for the inconsistency!